site/docs/v1/tech/apis/_application-response-body-base.adoc

```diff
@@ -80,6 +80,16 @@ The public key used when an `RSA` signing algorithms has been selected. The publ
 +
 [deprecated]#Removed in version 1.6.0# In version 1.6.0 and beyond, JWT configuration can be managed in the `link:/docs/v1/tech/apis/keys[Keys API]` and Keymaster.
 
+[field]#{base_field_name}.jwtConfiguration.refreshTokenExpirationPolicy# [type]#[String]#::
+The refresh token expiration policy. The following are valid values:
+
+* `Fixed` - the expiration is calculated from the time the token is issued.
+* `SlidingWindow` - the expiration is calculated from the last time the token was used.
+* `SlidingWindowWithMaximumLifetime` - the expiration is calculated in the same manner as `SlidingWindow`, however, the token will eventually expire once the maximum lifetime is reached.
+
+[field]#{base_field_name}.jwtConfiguration.refreshTokenSlidingWindowConfiguration.maximumTimeToLiveInMinutes# [type]#[Integer]# [since]#Available since 1.46.0#::
+When renewing a Refresh Token in a Sliding Window, the maximum lifetime of a Refresh Token. When this value is reached, the Refresh Token is expired, regardless if a sliding window configuration is used. Value must be greater than 0.
```

Comment:
I think re-ordering the clauses of the first "sentence" makes it easier to parse. And I added "absolute". i.e.

_The absolute maximum lifetime of a Refresh Token when renewing a Refresh Token in a Sliding Window._